app/services/coronavirus/pages/sub_section_processor.rb

```diff
@@ -4,50 +4,60 @@ def self.call(*args)
       new(*args).output
     end
 
-    attr_reader :sub_sections, :action_link
+    attr_reader :sub_sections
 
     def initialize(sub_sections)
       @sub_sections = [sub_sections].flatten
```

Comment:
Do you know what the purpose of this convert to array and flatten is? I'm somewhat guessing it's so this can optionally take an array or a single object but this seems an unusual way of doing it (`Array(sub_sections)` would be more conventional) and it doesn't seem to be tested. I'm not sure if it's needed? Presumably this never gets called without an array of sub_sections?